specs/cli/cli.md

```diff
@@ -0,0 +1,56 @@
+# FAST CLI (WIP)
+
+## Overview
+FAST CLI is a set of tools used for creating, building and developing FAST projects.
+
+FAST CLI will have an independent code base and will be packaged and installable on any .
+
+## Use Cases
+- A Front-End Engineer can use FAST CLI to generate a new FAST design system project with a single command.
```

Comment:
```suggestion
- A user can use FAST CLI to generate a new FAST design system project with a single command.
```

"Front-End Engineer" seems un-necessarily specific and narrow to me, any reason this can't just be "user"?